docs/user_guide/request_cancellation.md

```diff
@@ -49,8 +55,26 @@ for more details on how to issue requests from the client-side.
 See gRPC guide on RPC [cancellation](https://grpc.io/docs/guides/cancellation/) for
 finer details.
 
+## Handling in Triton Core
+
+Triton core checks for requests that have been cancelled at some critical points
+when using [dynamic](./model_configuration.md#dynamic-batcher) or
+[sequence batching](./model_configuration.md#sequence-batcher). We also test for
+the cancelled requests after every [ensemble](./model_configuration.md#ensemble-scheduler)
+step and terminate further processing the requests.
+
+On detecting a cancelled request, Triton core responds with CANCELLED status. If a request
+is cancelled when using [sequence_batching](./model_configuration.md#sequence-batcher),
+then all the pending requests in the same sequence will also be cancelled. The sequence
+is represented by the requests that has identical sequence id.
+
+**Note**: Currently, Triton core does not detect cancellation status of a request once
+it is forwarded to [rate limiter](./rate_limiter.md). Improving the request cancellation
+detection and handling within Triton core is work in progress.
+
+## Handling in Backend
 
-Upon receiving request cancellation, triton does its best to cancel request
+Upon receiving request cancellation, triton does its best to terminate request
```

Comment:
Double check how the bullet points below are rending, might be wrong

Reply:
Good catch! Fixed,